src/components/DeliveryDetails/RecipientForm/RecipientForm.tsx

```diff
@@ -112,14 +116,20 @@ const RecipientForm = () => {
           value={message.value}
         />
       </MessageWrapper>
-      <InfoText>
-        {t(
-          'recipientForm.info-text',
-          'To edit your gift delivery details, access MyAccount and click on the corresponding transaction.'
-        )}
-      </InfoText>
+      {!isMyAccount && (
+        <InfoText>
+          {t(
+            'recipientForm.info-text',
+            'To edit your gift delivery details, access MyAccount and click on the corresponding transaction.'
+          )}
+        </InfoText>
+      )}
     </StyledRecipientForm>
   );
 };
 
+RecipientForm.defaultProps = {
```

Comment:
should be deleted - defaultProps are part of prop types which we are not using anymore